.github/workflows/reusable_docker_build.yml

```diff
@@ -27,14 +27,21 @@ on:
 
 permissions: {}
 
+concurrency:
+  group: 'docker_build-${{ github.ref }}'
+  cancel-in-progress: true
+
 jobs:
   docker:
-    name: 'Build and push Docker container'
+    name: Build and push Docker container
     runs-on: ubuntu-latest
     timeout-minutes: 90
+    permissions:
+      contents: read
+      packages: write
 
     steps:
-      - name: 'Check out Dockerfile'
+      - name: Checkout the repo
```

Comment:
Note that it only checks out a single Dockerfile from the specified repo

Reply:
Got it. Reverted in https://github.com/mesh-adaptation/docs/pull/159/commits/ca08eece5e1a178d123f1c929a479804b5ee6d7e.